.travis.yml

```diff
@@ -19,6 +19,11 @@ matrix:
       env:
         - OS_TYPE=doxygen
       addons: { apt: { packages: ["cmake", "graphviz"] } }
+    - os: linux
+      dist: xenial
```

Comment:
if you don't need to force `xenial`, drop it;
at this point in time, this should be the default docker image